src/applications/vaos/services/mocks/index.js

```diff
@@ -278,6 +279,15 @@ const responses = {
       data: facilitiesV2.data.find(facility => facility.id === req.params.id),
     });
   },
+  'GET /vaos/v2/providers/:id': (req, res) => {
+    return res.json({
+      data: {
+        attributes: providersV2.data.find(
+          provider => provider.providerIdentifier === Number(req.params.id),
+        ),
+      },
+    });
+  },
```

Comment:
Any reason for the difference from the others? The provider.json file should specify the mock in the JSONApi format.

Reply:
I left this unchanged per our discussion yesterday, let me know if you still feel we need to do additional formatting.